packages/typeorm/src/utils.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2017,2020. All Rights Reserved.
+// Node module: @loopback/typeorm
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {JsonSchemaOptions, SchemaRef} from '@loopback/rest';
```

Comment:
❗ `@loopback/typeorm` should not require consumers to use `@loopback/rest`.

At minimum, change this line to import OpenAPI types from `@loopback/openapi-v3`.

As I mentioned above, I think it would be best to move OpenAPI-related parts into a new package, but I guess that can be done in a follow-up pull request.

Reply:
Note to self: Create follow up task.